.circleci/config.yml

```diff
@@ -48,7 +48,6 @@ jobs:
                 name: Create env file
                 command: |
                   touch .env
-                  echo 'POSTGRES_URL=postgres://postgres:example@db.local:5432/network_adequacy\nPOSTGRES_URL_POSTGRES=postgres://postgres:example@db.local:5432/postgres' > .env
```

Comment:
So where are we pulling the .env info from?

Reply:
Is it actually using the same machine for sure?

Reply:
No, the two builds run in different VMs. The frontend build/test process doesn't actually need any env vars though which is why I have removed this.